src/dom.js

```diff
@@ -156,6 +184,15 @@ export function childElement(parent, callback) {
  * @return {?Element}
  */
 export function childElementByAttr(parent, attr, opt_value) {
+  if (scopeSelectorSupported == null) {
+    scopeSelectorSupported = isScopeSelectorSupported();
+  }
+  if (scopeSelectorSupported) {
+    if (opt_value !== undefined) {
+      return parent.querySelector(':scope [' + attr + '="' + opt_value + '"]');
```

Comment:
We never actually look by value. Should I just delete that code?

I benchmarked this, by the way, and it was consistently 2x faster.